x-pack/legacy/plugins/ml/public/services/ml_api_service/index.d.ts

```diff
@@ -92,6 +99,7 @@ declare interface Ml {
   getJobStats(obj: object): Promise<any>;
   getDatafeedStats(obj: object): Promise<any>;
   esSearch(obj: object): any;
+  esSearchRx(obj: object): Observable<any>;
```

Comment:
Should we name this `esSearch$`?

Reply:
done in https://github.com/elastic/kibana/pull/51008/commits/c72dddf5ed6d8184db3ed0ea95cc1850b83ab701